### PR TITLE
ticket 3 - get topics complete

### PR DESCRIPTION
**Pull request by gsaltuk**
Unsure on error handling - added a default 404 however this is handled automatically I believe?

### CODE REVIEW

**Review by yalicee — CHANGES_REQUESTED** [review]
Looks great but please remove all of your console.logs before you push to github from now on. Also please handle the 404 manually.

**Comment by gsaltuk** [comment]
Manual error handling now added and console.logs removed

File: app.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controllers");
+const app = express();
+
+app.use(express.json());
```
Comment (Geo-Beard) [inline app.js]: Remove app.use(express.json()) - you don't need it yet, so it's outside the scope of this ticket.

**Review by Geo-Beard — COMMENTED** [review]
Looks good overall - just one note on using app.use(express.json()) - need to remove it until you need it.

**Pull request merged**